packages/thirdweb/package.json

```diff
@@ -147,6 +147,7 @@
     "fuse.js": "7.0.0",
     "mipd": "0.0.5",
     "qrcode": "1.5.3",
+    "secrets.js-34r7h": "2.0.2",
```

Comment:
consider using https://github.com/privy-io/shamir-secret-sharing instead, has more recent audits and also is half the size + more modern codebase (ts, tree shaking works etc)

Reply:
alternatively we can also write it ourselves and get it audited